buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -203,6 +210,10 @@
       "value": "$(Build.BuildNumber)",
       "allowOverride": true
     },
+    "PB_SkipTests": {
+      "value": "false",
+      "allowOverride": false
```

Comment:
Is 'false' intentional?  Temporary until something else happens?

Reply:
No typo. Thank you for catching